src/es/components/organisms/header/default-/default-.css

```diff
@@ -38,7 +38,8 @@
   --header-default-logo-width: var(--header-default-logo-width-custom, 90px);
   --header-default-m-navigation-background-color-mobile: var(--background-color);
   --header-default-m-navigation-height-mobile: calc(100vh - var(--header-default-m-navigation-top-mobile));
-  --header-default-m-navigation-height-open-mobile: var(--header-default-m-navigation-height-mobile);
```

Comment:
Is this working without side effect on the existing navigation? Did you test it with mpro or alna?

Reply:
@Weedshaker Accidentally removed, other components need this variable, i added back.